types/terser-webpack-plugin/index.d.ts

```diff
@@ -49,6 +49,6 @@ declare namespace TerserPlugin {
         | RegExp
         | ExtractCommentFn
         | ExtractCommentOptions;
-        warningsFilter?: (warning: any, source: any) => boolean;
+        warningsFilter?: (warning: string, source: string, file: string) => boolean;
```

Comment:
According to the documentation the `source` argument can be undefined if sourcemaps are not used. So this should be included in the type definition or 'strictNullChecks' are not able to catch this pitfall.